_artifacts/opensearch-drivers/opensearch-odbc-1.5.0.0-macos-x64.markdown

```diff
@@ -0,0 +1,12 @@
+---
+role: driver
+artifact_id: opensearch-sql-odbc
+version: 1.5.0.0
+platform: macos
+architecture: x64
+artifact_url: https://artifacts.opensearch.org/opensearch-clients/odbc/opensearch-sql-odbc-driver-64-bit-1.5.0.0-Darwin.pkg
+slug: opensearch-sql-odbc-mac-1.5.0.0
+category: opensearch
+type: pkg
+code_signer: true
```

Comment:
Should this be `mac_signer` since windows has `windows_signer`?

Reply:
The signing type is Windows listed [here](https://github.com/opensearch-project/opensearch-build/blob/main/README.md#windows), so I would keep it as `code_signer`.